pkg/api/validation/validation.go

```diff
@@ -1039,6 +1039,14 @@ func ValidateService(service *api.Service) errs.ValidationErrorList {
 	if len(service.Spec.Ports) == 0 {
 		allErrs = append(allErrs, errs.NewFieldRequired("spec.ports"))
 	}
+	if service.Spec.Type == api.ServiceTypeLoadBalancer {
```

Comment:
Preventing someone from explicitly choosing nodeport=10250 is orthogonal if we aren't already doing it